### PR TITLE
Add README content for creating and deploying a function

### PR DESCRIPTION
**Pull request by frostie**
This covers 4 README updates (Java, Go, Node.js, and Python) for the Cloud Functions integration insiders release.

### CODE REVIEW

File: java/cloud-function-hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+For details on how to use this sample as a template in Cloud Code, see the [Create and deploy a function](https://cloud.google.com/code/docs/vscode/create-deploy-function) quickstart for VS Code.
+
+## Table of Contents
+
+* [Directory contents](#directory-contents)
+* [Getting started with VS Code](#getting-started-with-vs-code)
+* [Sign up for user research](#sign-up-for-user-research)
+
+## Directory contents
+* `launch.json` - the required configurations for your function
```
Comment (glouischandra) [inline java/cloud-function-hello-world/README.md]: nit: "required Cloud Code configurations"

**Review by glouischandra — APPROVED** [review]

File: java/cloud-function-hello-world/README.md (hunk 1 above)
Comment (frostie) [reply java/cloud-function-hello-world/README.md]: Done. Thanks!

**Pull request merged**